engines/interpolantmc.cpp

```diff
@@ -120,26 +124,42 @@ bool InterpolantMC::step(int i)
   }
 
   Term bad_i = unroller_.at_time(bad_, i);
-  bad_disjuncts_ = solver_->make_term(Or, bad_disjuncts_, bad_i);
+  if (interp_props_ == InterpPropsEnum::ONLY_LAST || i == 1) {
```

Comment:
first should be i == 0

Reply:
The initial/zeroth time step (i == 0) is handled by `step_0()`:

```c
  if (i == 0) {
    // Can't get an interpolant at bound 0
    // only checking for trivial bug
    return step_0();
  }
```

The "first" here refers to the states reached after "one" step from the initial state.